build-artifacts.py

```diff
@@ -15,16 +13,14 @@
 
 change = sys.argv[1]
 
-config = configparser.ConfigParser()
-conf_path =  os.path.expanduser("~/.config/oci.conf")
-config.read(conf_path)
+cfg = config.load("~/.config/oci.conf")
```

Comment:
I think we should let config module load the default file. The only reason
to use special path is when a user ask to do this.